cartridges/app_storefront_training/cartridge/static/default/css/cart.css

```diff
@@ -0,0 +1,5 @@
+.hidden-promo {
```

Comment:
Don't commit the built files like .css and .css.map. When the branch reaches the development environment its own build is triggered